ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/MultiTenancyUserUtil.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.api.jsonrpc.internal.privacy.methods;
+
+import java.util.Optional;
+
+import io.vertx.ext.auth.User;
+
+public class MultiTenancyUserUtil {
+  private static final String PRIVACY_PUBLIC_KEY = "privacyPublicKey";
```

Comment:
what does this represent? I.e. guess its a fieldname in a principal right? Do we want to name it as such - eg enclavePublicKeyFieldName?